lib/general-calendar/proper-of-time.ts

```diff
@@ -537,22 +537,22 @@ export class ProperOfTime extends CalendarDef {
       // Sunday of the Word of God
       // http://www.vatican.va/content/francesco/en/motu_proprio/documents/papa-francesco-motu-proprio-20190930_aperuit-illis.html
       if (week === 3 && dow === 0) {
-        this.#newLiturgicalDayDef(`sunday_of_the_word_of_god`, {
+        this.#newLiturgicalDayDef('sunday_of_the_word_of_god', {
           ...data,
-          i18nDef: [`names:sunday_of_the_word_of_god`],
+          i18nDef: ['names:sunday_of_the_word_of_god'],
         });
-      }
+      } // eslint-disable-line @typescript-eslint/brace-style
 
       // 34th week in Ordinary Time,
       // or the Solemnity of our Lord Jesus Christ, King of the Universe.
       else if (week === 34 && dow === 0) {
```

Comment:
In order to make the `@typescript-eslint/brace-style` rule happy, we need to place `}` from L544 to L548 (which makes sense to me).

```suggestion

        // 34th week in Ordinary Time,
        // or the Solemnity of our Lord Jesus Christ, King of the Universe.
      } else if (week === 34 && dow === 0) {
```

Reply:
ok